subprojects/packagefiles/fdk-aac/meson.build

```diff
@@ -0,0 +1,403 @@
+  'libSACdec/src/sac_dec.h',
+  'libSACdec/src/sac_dec_conceal.h',
+  'libSACdec/src/sac_dec_interface.h',
+  'libSACdec/src/sac_dec_ssc_struct.h',
+  'libSACdec/src/sac_process.h',
+  'libSACdec/src/sac_qmf.h',
+  'libSACdec/src/sac_reshapeBBEnv.h',
+  'libSACdec/src/sac_rom.h',
+  'libSACdec/src/sac_smoothing.h',
+  'libSACdec/src/sac_stp.h',
+  'libSACdec/src/sac_tsd.h',
+  'libSACenc/src/sacenc_bitstream.h',
+  'libSACenc/src/sacenc_const.h',
+  'libSACenc/src/sacenc_delay.h',
+  'libSACenc/src/sacenc_dmx_tdom_enh.h',
+  'libSACenc/src/sacenc_filter.h',
+  'libSACenc/src/sacenc_framewindowing.h',
+  'libSACenc/src/sacenc_huff_tab.h',
+  'libSACenc/src/sacenc_nlc_enc.h',
+  'libSACenc/src/sacenc_onsetdetect.h',
+  'libSACenc/src/sacenc_paramextract.h',
+  'libSACenc/src/sacenc_staticgain.h',
+  'libSACenc/src/sacenc_tree.h',
+  'libSACenc/src/sacenc_vectorfunctions.h',
+  'libSBRdec/src/env_calc.h',
+  'libSBRdec/src/env_dec.h',
+  'libSBRdec/src/env_extr.h',
+  'libSBRdec/src/hbe.h',
+  'libSBRdec/src/HFgen_preFlat.h',
+  'libSBRdec/src/huff_dec.h',
+  'libSBRdec/src/lpp_tran.h',
+  'libSBRdec/src/psbitdec.h',
+  'libSBRdec/src/psdec.h',
+  'libSBRdec/src/psdec_drm.h',
+  'libSBRdec/src/pvc_dec.h',
+  'libSBRdec/src/sbr_deb.h',
+  'libSBRdec/src/sbr_dec.h',
+  'libSBRdec/src/sbr_ram.h',
+  'libSBRdec/src/sbr_rom.h',
+  'libSBRdec/src/sbrdec_drc.h',
+  'libSBRdec/src/sbrdec_freq_sca.h',
+  'libSBRdec/src/transcendent.h',
+  'libSBRenc/src/bit_sbr.h',
+  'libSBRenc/src/cmondata.h',
+  'libSBRenc/src/code_env.h',
+  'libSBRenc/src/env_bit.h',
+  'libSBRenc/src/env_est.h',
+  'libSBRenc/src/fram_gen.h',
+  'libSBRenc/src/invf_est.h',
+  'libSBRenc/src/mh_det.h',
+  'libSBRenc/src/nf_est.h',
+  'libSBRenc/src/ps_bitenc.h',
+  'libSBRenc/src/ps_const.h',
+  'libSBRenc/src/ps_encode.h',
+  'libSBRenc/src/ps_main.h',
+  'libSBRenc/src/resampler.h',
+  'libSBRenc/src/sbr.h',
+  'libSBRenc/src/sbr_def.h',
+  'libSBRenc/src/sbr_misc.h',
+  'libSBRenc/src/sbrenc_freq_sca.h',
+  'libSBRenc/src/sbrenc_ram.h',
+  'libSBRenc/src/sbrenc_rom.h',
+  'libSBRenc/src/ton_corr.h',
+  'libSBRenc/src/tran_det.h',
+  'libPCMutils/src/version.h'
+)
+
+fdk_aac_headers = [
+  'libAACdec/include/aacdecoder_lib.h',
+  'libAACenc/include/aacenc_lib.h',
+  'libSYS/include/FDK_audio.h',
+  'libSYS/include/genericStds.h',
+  'libSYS/include/machine_type.h',
+  'libSYS/include/syslib_channelMapDescr.h',
+]
+
+install_headers(files(fdk_aac_headers), subdir : 'fdk-aac')
+
+# Make using fdk-aac as a subproject behave the same as when used installed.
+subdir('fdk-aac')
+
+cc = meson.get_compiler('c')
+m_dep = cc.find_library('m', required : false)
+
+extra_args = []
+if get_option('remove-date').enabled()
+  extra_args += ['-DSUPPRESS_BUILD_DATE_INFO']
+endif
+
+fdk_aac = library(
+  'fdk_aac',
+  fdk_aac_src,
+  dependencies: [m_dep],
+  include_directories : fdk_aac_inc,
+  install : true,
+  cpp_args : ['-fno-exceptions', '-fno-rtti', '-Dfdk_aac_EXPORTS'] + extra_args,
```

Comment:
For msvc you probably need to use cc.get_supported_arguments()

Reply:
cl : Command line warning D9002 : ignoring unknown option '-fno-exceptions'
cl : Command line warning D9002 : ignoring unknown option '-fno-rtti'

Reply:
Thanks! Fixed.